pkg/minishift/update/update.go

```diff
@@ -282,11 +282,11 @@ func updateBinary(binary string) error {
 		Hash: crypto.SHA256,
 	})
 	if err != nil {
-		fmt.Errorf("Cannot apply binary update: %s", err)
-		err := update.RollbackError(err)
-		if err != nil {
-			return errors.New(fmt.Sprintf("Failed to rollback update: %s", err))
+		rollbackErr := update.RollbackError(err)
+		if rollbackErr != nil {
```

Comment:
So how does this fix things? Even in the old code _RollbackError_ is called. I have a hard time seeing the difference. Can you explain. I get that the _fmt.Errorf_ is bogus, but how does this fix the issue.

With this in place what happens now when the binary is owned by root?